README.md

```diff
@@ -39,6 +39,41 @@ This module is a replacement for the [puppet_metrics_dashboard module](https://f
 
 ## Setup
 
+### Firewall Requirements for Puppet Operational Dashboards
+
+To ensure smooth communication between services in the Puppet Operational Dashboards module, the following firewall rules must be configured:
+
+| Application          | Port  | Protocol | Deployment Notes                                                                 |
+|----------------------|-------|----------|----------------------------------------------------------------------------------|
+| **Telegraf**         | 8086  | TCP      | Connects to InfluxDB for metric collection.                                      |
+| **InfluxDB**         | 8086  | TCP      | Receives metrics from Telegraf.                                                 |
+| **Grafana**          | 3000  | TCP      | Used for accessing the dashboard interface.                                      |
+| **PostgreSQL**       | 5432  | TCP      | Required for metrics collection in both Puppet Enterprise and Open Source setups.|
```

Comment:
Metrics are collected _from_ Postgres but it is not serving as a storage location for dashabord data, so not a hard requirement